salt/modules/parted_partition.py

```diff
@@ -612,20 +612,17 @@ def rescue(device, start, end):
     return out
 
 
-def resize(device, minor, start, end):
+def resize(device, minor, end):
```

Comment:
We cannot remove an arg without properly deprecating it, and in this instance we do not want to deprecate it as we still want users to use older versions of parted. Can you create a function that detects the version we are on and based on the version change the command structure. I also noticed that `parted resize` was actually removed (https://www.gnu.org/software/parted/manual/html_node/Command-explanations.html#Command-explanations) so that probably needs to be updated.

This will also require tests